src/redux/walletconnect.ts

```diff
@@ -149,6 +149,8 @@ export interface WalletconnectApprovalSheetRouteParams {
   receivedTimestamp: number;
   meta?: {
     chainId: number;
+    chainIds?: number[];
```

Comment:
Fair point. Looks like some legacy holdover since we might get `chainId` from `params` or `params.meta`. Seems to be isolated now, but at a glance I'd need to do some fussing with Typescript too so I'd like to keep it out of this PR. I've added a follow-up for myself 👍